buildenv/jenkins/docker-slaves/x86/centos6.9/Dockerfile

```diff
@@ -126,8 +126,8 @@ RUN cd /etc/yum.repos.d \
   && ln -s /opt/rh/devtoolset-2/root/usr/bin/gcc-4.8 /usr/bin/cc \
   && yum -y update \
   && yum -y install centos-release-scl \
-  && yum -y install devtoolset-7-gcc devtoolset-7-binutils \
-  && yum -y install devtoolset-7-gcc-c++ devtoolset-2-gcc-gfortran \
+  && yum -y install devtoolset-7-gcc-7.3* devtoolset-7-binutils \
```

Comment:
Similar comment

Reply:
I saw multiple versions of gcc-7.3 and gcc-7.2 in that repo so I think that it will remain in that repo.

Reply:
It is similar to what Adopt is doing